chapters/tests/sections/manual_tests.tex

```diff
@@ -0,0 +1,58 @@
+\section{Manual testing} \label{sec:manual_testing}
+Manual testing is responsible for testing the game as a whole.
+It tests the results and not particular methods or parts of the code.
+Manual tests scenarios were created that describe in detail what should happen in the game after some actions are performed.
+This tests cover the entirety of the application and can be viewed at \url{https://github.com/Non-Euclidean-World/HyperTesting}.
+The tests are written as Markdown files that have references to image and video files that show the expected behavior of the game.
+The test cases are divided into four parts: title, description, prerequisites and steps.
+The title describes the name of the test case.
+The description describes what the test case is supposed to test.
+The prerequisites describe what has to be done before the test case can be performed.
+The steps is a table the describes the action that has to be performed and the expected result.
+Below is an example test case. 
+It is important to note that the test case has been modified to fit the paper format by changing links to the images into references.
+
+\todo{Might want to write a specific test case to put here since all the nice ones have videos in them}
+
+\subsection*{Flashlight works}\label{flashlight-works}
```

Comment:
there's something like minipage environment we could use that, and maybe put this in a box of some sort to show that this is a test case and a part of the text itself so to speak